GitUI/HelperDialogs/FormRemoteProcess.cs

```diff
@@ -26,42 +26,33 @@ public partial class FormRemoteProcess : FormProcess
             new TranslationString("Host Fingerprint not registered");
         #endregion
 
-        public bool Plink { get; set; }
         private bool _restart;
-        protected readonly GitModule Module;
-
-        // only for translation
-        protected FormRemoteProcess()
-        {
-        }
-
-        public FormRemoteProcess(GitModule module, string process, ArgumentString arguments)
-            : base(process, arguments, module.WorkingDir, null, true)
-        {
-            Module = module;
-        }
+        private string _urlTryingToConnect = string.Empty;
 
-        public FormRemoteProcess(GitModule module, ArgumentString arguments)
-            : base(null, arguments, module.WorkingDir, null, true)
+        [Obsolete("For VS designer and translation test only. Do not remove.")]
+        public FormRemoteProcess()
```

Comment:
Why public? Reflection does not need it.

Reply:
I couldn't get translations working, probably can be returned to `private protected`

Reply:
reverted